tests/SpeakerServiceBulkSendChunkingTest.php

```
@@ -0,0 +1,369 @@
+
+        $presentation = new \models\summit\Presentation();
+        self::$summit->addEvent($presentation);
+        $presentation->setTitle("Multi-page chunk test {$firstName}");
+        $presentation->setAbstract("Abstract {$firstName}");
+        $presentation->setCategory(self::$defaultTrack);
+        $presentation->setType(self::$defaultPresentationType);
+        $presentation->setStartDate(new \DateTime('now', new \DateTimeZone('UTC')));
+        $presentation->setEndDate(new \DateTime('+1 hour', new \DateTimeZone('UTC')));
+
+        $seeded = [];
+        for ($i = 0; $i < SpeakerService::CHUNK_SIZE + 1; $i++) {
```

PHPStan (2.2.8)

[error] Access to undefined constant services\model\SpeakerService::CHUNK_SIZE.


(classConstant.notFound)